src/org/zaproxy/zap/extension/ascan/ActiveScan.java

```diff
@@ -83,8 +84,8 @@
 	private static final Logger log = Logger.getLogger(ActiveScan.class);
 
 	public ActiveScan(String displayName, ScannerParam scannerParam, 
-			ConnectionParam param, ScanPolicy scanPolicy) {
-		super(scannerParam, param, scanPolicy);
+			ConnectionParam param, ScanPolicy scanPolicy, RuleConfigParam ruleConfigParam) {
+		super(scannerParam, param, scanPolicy, ruleConfigParam);

```

Comment:
Is this likely to cause backward compatibility issues? Shouldn't a new constructor be added and the old one (to be marked deprecated) pass a null or default ruleConfigParam to it?